content/applications/productivity/discuss/advanced/email_servers.rst

```diff
@@ -224,6 +224,20 @@ alias in your mail server.
 
 .. _Office 365 documentation:
     https://support.office.com/en-us/article/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-office-365-69f58e99-c550-4274-ad18-c805d654b4c4
+    
+Set up different outgoing servers in a multi-company environment
+================================================================
+
+The "From Filter" allows to use a specific outgoing depending of the "From". If an email is sent from Odoo, the email server will be chosen as follows:
+
+#.  Odoo searches for an email server having the same "From Filter" as the "From" defined in the outgoing email. For example, if the "From" is foo@bar.com, only the email servers having the "From Filter" equals to foo@bar.com will be returned.
```

Comment:
`Odoo searches for an email server having the same "From Filter" as the "From" email address defined in the outgoing email. For example, if the "From" email address is test@example.com, only the email servers having the "From Filter" equal to test@example.com will be returned. `